src/tokens.ts

```diff
@@ -0,0 +1,7 @@
+import { Token } from '@lumino/coreutils';
+import { IWidgetTracker } from '@jupyterlab/apputils';
+import { ImageEditorDocumentWidget } from './editor';
+
+
+type IImageEditorTracker = IWidgetTracker<ImageEditorDocumentWidget>;
```

Comment:
You will need to export this type too:

```suggestion
export type IImageEditorTracker = IWidgetTracker<ImageEditorDocumentWidget>;
```